dotnet/src/SemanticKernel.Skills/Skills.Memory.VectorDB/Qdrant/DotNet/Internal/Diagnostics/Exception.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+
+namespace Qdrant.DotNet.Internal.Diagnostics;
```

Comment:
Lots of namespace issues. This will need to be standardized everywhere... I'm not going to call out each one, as they've already been pointed out in previous PRs.

Reply:
@shawncal these were address in other files per other PR copying these files back over to ensure this PR is updated correctly.